jboss/container/maven/8.2.3.6.8/configure.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -e
+
+# This file is shipped by a Maven package and sets JAVA_HOME to
+# an OpenJDK-specific path. This causes problems for OpenJ9 containers
+# as the path is not correct for them.  We don't need this in any of
+# the containers because ew set JAVA_HOME in the container metadata.
+# Blank the file rather than removing it, to avoid a warning message
+# from /usr/bin/mvn.
+if [ /etc/java/maven.conf ]; then
```

Comment:
This still doesn't look right.

Reply:
You're right, it isn't :( I've fixed it up this morning, built and run tests over the JDK11 image .

Reply:
OK. Looks good now.